ai/neuronal_network/compute_accuracy.py

```diff
@@ -0,0 +1,81 @@
+import json
+import numpy as np
+from keras.models import load_model
+import random
+from ai.neuronal_network.model_builder import format_json_match, purge_data, format_matchs
+
+data = json.load(open('./trainning_data/trainning_data.json', 'r'))
```

Comment:
This is directly linked with my tasks about implement the AI in rust, but can you add absolute paths whenever you open a file ? Like getting cwd and concatenate cwd + "/ai/{dir}/{file_name}.."
Because the rust crate available for the conversion will always use python as module from the root of the project.